src/pages/AdminDashboard/CreateRoomForm/CreateRoomForm.tsx

```diff
@@ -237,13 +173,7 @@ const CreateRoomForm = ({ fields, onFormSubmit, onCreateFormReset }) => {
         <Formik
             enableReinitialize
             initialValues={fields}
-            validate={(values) => {
-                const errors: Partial<Room> = {}
-                if (!values.name) {
-                    errors.name = 'Required'
-                }
-                return errors
-            }}
+            //validationSchema={validationSchema}
```

Comment:
comment needed?

Reply:
@ntchou will take on with form validation later

Reply:
i did some tests and i forgot